Neos.Neos/Documentation/References/NeosFusionReference.rst

```diff
@@ -323,23 +323,23 @@ When used as process the script tag is appended to the processed value.
 
 Example::
 
-  renderer.@process.debug = Neos.Fusion:Debug.Console {
+  renderer.@process.debug = Neos.Fusion:DebugConsole {
```

Comment:
But the prototype is called `Neos.Fusion:Debug.Console`, I think only the headline is incorrect in this doc.

Reply:
lol now i mixed it up :D